dashboard/dashboard_functions.py

```diff
@@ -16,10 +16,13 @@ def make_date_radio():
     return radio
 
 
-def make_date_filter(df: pd.DataFrame, radio: str):
-    """Makes a date filter."""
-    min_date = min(df['at'].dt.date)
-    max_date = max(df['at'].dt.date)
+
+def make_date_filter(scrape_df: pd.DataFrame, interaction_df: pd.DataFrame, radio: str):
+    """Makes a date filter"""
+    min_date = min([min(scrape_df['scrape_at'].dt.date),
+                   min(interaction_df['interact_at'].dt.date)])
+    max_date = max([max(scrape_df['scrape_at'].dt.date),
+                   max(interaction_df['interact_at'].dt.date)])
```

Comment:
very nice, now you have two so you need the minimum of each minimum and vice versa for the maximum date